### ISSUE TITLE
Multi Touch Gesture not firing slideChange on Windows 10

### ISSUE THREAD

**Issue by ChiaraTo** [nolimits4web/swiper#6175]
### Check that this is really a bug

- [X] I confirm

### Reproduction link

https://studio.swiperjs.com/play

### Bug description

In my project I use the current version of the library (8.4.4). On Linux devices with touch capabilities I can swipe horizontally with one or more fingers, on the Windows devices only with one. With the 4th Version of the library it was possible to swipe with multiple fingers on Windows. 
I noticed the same behavior testing the Swiper Demo https://studio.swiperjs.com/play on a Windows 10 device.
#### My findings:
On Windows as soon as two or more fingers are used to swipe, the touchstart event is fired twice or more (one for every finger), but the touchend is fired only once (for the first finger) and no slideChange event is fired.
As a workaround I tried following code but the swiper.touches.diff parameter is not reliable to check the swipe direction.

```
onTouchStart={(swiper, event) => {
  const {activeIndex} = swiper;
  const {diff} = swiper.touches;
  if (event.touches.length > 1 && activeIndex !== 0) {
    if (diff < 0) { // right direction
      swiper.slideTo(activeIndex + 1, 1000);
    } else { // left direction
      swiper.slideTo(activeIndex - 1, 1000);
    }                           
  }
}}
```
I also tried this approach https://github.com/nolimits4web/swiper/issues/3728 but without success.

Any hints?
Thank you in advance

### Expected Behavior

On Windows 10 devices with touch capabilities the horizontal swiping is possible also with multiple fingers pan gestures.

### Actual Behavior

On Windows 10 devices with touch capabilities the horizontal swiping is possible only with one finger pan gesture.

### Swiper version

8.4.4

### Platform/Target and Browser Versions

Windows, all Browsers

### Validations

- [X] Follow our [Code of Conduct](https://github.com/nolimits4web/swiper/blob/master/CODE_OF_CONDUCT.md)
- [X] Read the [docs](https://swiperjs.com/swiper-api).
- [X] Check that there isn't already an issue that request the same feature to avoid creating a duplicate.
- [X] Make sure this is a Swiper issue and not a framework-specific issue

### Would you like to open a PR for this bug?

- [ ] I'm willing to open a PR

**Comment by nolimits4web** [comment]
Issue is closed because of outdated/irrelevant/not actual/needed more information/inactivity.

If this issue is still actual and reproducible for latest version of Swiper, please create new issue and fill the issue template correctly:
* Clearly describe the issue including steps to reproduce when it is a bug.
* Make sure you fill in the earliest version that you know has the issue.
* Provide live link or JSFiddle/Codepen or website with issue
